app/controllers/reviews_controller.rb

```diff
@@ -21,6 +21,14 @@ def edit
     @review = Review.find(params[:id])
   end
 
+  def destroy
+    review = Review.find(params[:id])
+    @show = review.show
+    Review.destroy(params[:id])
```

Comment:
Why set the variable on line 25 if you're not going to use it on line 27 (e.g. `review.destroy`)? Also, why not gate the `destroy` action the same way you would a `save` or `update`, with some `if... then` style flow control? That way if for any reason something goes wrong with the deletion, you can handle it gracefully.